# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libcinnamon-desktop/no-gir.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- cinnamon-desktop-6.2.0/libcinnamon-desktop/meson.build.vanilla	2024-09-06 22:41:20.491237325 +0200
+++ cinnamon-desktop-6.2.0/libcinnamon-desktop/meson.build	2024-09-06 22:41:27.652293910 +0200
@@ -92,32 +92,6 @@
   )
 endif
 
-cdesktop_gir = gnome.generate_gir(libcinnamon_desktop,
-  sources: 'cdesktop-enums.h',
-  namespace: 'CDesktopEnums',
-  nsversion: '3.0',
-  identifier_prefix: 'CDesktop',
-  symbol_prefix: 'c_desktop',
-  extra_args: [
-    '-DGNOME_DESKTOP_USE_UNSTABLE_API',
-  ],
-  install: true,
-)
-
-gnome.generate_gir(libcinnamon_desktop,
-  sources: libcinnamon_desktop_gir_sources + libcinnamon_desktop_headers,
-  namespace: 'CinnamonDesktop',
-  nsversion: '3.0',
-  export_packages: 'cinnamon-desktop',
-  identifier_prefix: 'Gnome',
-  symbol_prefix: 'gnome_',
-  includes: ['GObject-2.0', 'Gtk-3.0', cdesktop_gir[0], ],
-  extra_args: [
-    '-DGNOME_DESKTOP_USE_UNSTABLE_API',
-  ],
-  install: true,
-)
-
 # FIXME
 # https://github.com/mesonbuild/meson/issues/1687
 custom_target('gsettings-enums',
--- cinnamon-desktop-6.2.0/libcvc/meson.build.vanilla	2024-09-06 22:41:52.445489754 +0200
+++ cinnamon-desktop-6.2.0/libcvc/meson.build	2024-09-06 22:42:03.703578650 +0200
@@ -59,20 +59,6 @@
   version: '0.0.0',
 )
 
-gnome.generate_gir(libcvc,
-  sources: libcvc_gir_sources + libcvc_headers + libcvc_enums,
-  namespace: 'Cvc',
-  nsversion: '1.0',
-  identifier_prefix: 'Gvc',
-  symbol_prefix: 'gvc_',
-  includes: [
-    'GObject-2.0',
-    'Gio-2.0',
-  ],
-  install: true,
-)
-
-
 install_headers(
   libcvc_headers,
   subdir: join_paths('cinnamon-desktop', 'libcvc'),
